_data/banking.yml

```
@@ -597,6 +741,8 @@ websites:
       - proprietary
       - hardware
     doc: https://www1.firstdirect.com/help/secure-key/
+    regions:
+      - us
```

```suggestion
      - gb
```
First Direct is part of HSBC in the UK.